server/tinylicious/src/routes/ordering/documents.ts

```diff
@@ -72,5 +81,33 @@ export function create(storage: IDocumentStorage): Router {
 		);
 	});
 
+	/**
+	 * Passes on content to all clients in a collaboration session happening on the document via means of signal.
+	 */
+	router.post("/:tenantId/:id/broadcast-signal", (request, response) => {
+		const tenantId = getParam(request.params, "tenantId");
+		const documentId = getParam(request.params, "id");
+		const signalContent = getParam(request.body, "signalContent");
+		const documentP = storage.getDocument(
+			getParam(request.params, "tenantId"),
+			getParam(request.params, "id"),
```

Comment:
Already extracted above
```suggestion
			tenantId,
			documentId,
```